src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+server.servlet.session.tracking-modes=COOKIE
```

Comment:
I can't remember what this means but i am sure this is wrong. No session should be created, no tracking is assumed. All information needed is supplied with every request.

Reply:
fix: remove